dowhy/causal_estimators/econml.py

```diff
@@ -180,8 +177,65 @@ def construct_symbolic_estimator(self, estimand):
     def shap_values(self, df: pd.DataFrame, *args, **kwargs):
         return self.estimator.shap_values(df[self._effect_modifier_names].values, *args, **kwargs)
 
+    def apply_multitreatment(self, df: pd.DataFrame, fun: Callable, *args, **kwargs):
+        ests = []
+        assert not isinstance(self._treatment_value, str)
+        assert is_sequence(self._treatment_value)
+
+        if df is None:
+            filtered_df = None
+        else:
+            filtered_df = df[self._effect_modifier_names].values
+
+        for tv in self._treatment_value:
+            ests.append(
+                fun(
+                    filtered_df,
+                    T0=self._control_value,
+                    T1=tv,
+                    *args,
+                    **kwargs,
+                )
+            )
+        est = np.stack(ests, axis=1)
+        return est
+
     def effect(self, df: pd.DataFrame, *args, **kwargs) -> np.ndarray:
-        return self.estimator.effect(df[self._effect_modifier_names].values, *args, **kwargs)
+        def effect_fun(filtered_df, T0, T1, *args, **kwargs):
+            return self.estimator.effect(filtered_df, T0=T0, T1=T1, *args, **kwargs)
+
+        return self.apply_multitreatment(df, effect_fun, *args, **kwargs)
+
+    def effect_interval(self, df: pd.DataFrame, *args, **kwargs) -> np.ndarray:
```

Comment:
will it possible to add a docstring for these methods? It will be good to add a docstring to the the methods we expect people to access. In the past, we were not so consistent, but this will be a good practice.